Cargo.toml

```diff
@@ -1,28 +1,28 @@
 [workspace]
 
 members = [
+    "pbt-gen",
+    "proto",
+    "tendermint",
+    "tools/proto-compiler",
+]
+
+exclude = [
```

Comment:
why do we exclude all the other members? won't we need those things for writing light node?